gbfs.md

```diff
@@ -345,11 +328,17 @@ The following fields are all attributes within the main `data` object for this f
 Field Name | REQUIRED | Type | Defines
 ---|---|---|---
 `system_id` | Yes | ID | This is a globally unique identifier for the vehicle share system. Each distinct system or geographic area in which vehicles are operated MUST have its own unique `system_id`. It is up to the publisher of the feed to guarantee uniqueness and MUST be checked against existing `system_id` fields in  [systems.csv](https://github.com/NABSA/gbfs/blob/master/systems.csv) to ensure this. This value is intended to remain the same over the life of the system. <br><br> System IDs SHOULD be recognizable as belonging to a particular system as opposed to random strings - for example, `bcycle_austin` or `biketown_pdx`.
-`language` | Yes | Language | The language that will be used throughout the rest of the files. It MUST match the value in the [gbfs.json](#gbfsjson) file.
-`name` | Yes | String | Name of the system to be displayed to customers.
-`opening_hours`<br/>*(added in v3.0-RC)*| Yes | String | Hours and dates of operation for the system in [OSM opening_hours](https://wiki.openstreetmap.org/wiki/Key:opening_hours) format. *(added in v3.0-RC)*
-`short_name` | OPTIONAL | String | OPTIONAL abbreviation for a system.
-`operator` | OPTIONAL | String | Name of the system operator.
+`languages` | Yes | Array | List of languages used in translated strings. Each element in the list must be of type Language.
+`name` | Yes | Array | Name of the system to be displayed to customers. An array with one object per supported language with the following keys:
+\-&nbsp; `text` | Yes | String | The translated text
+\-&nbsp; `language` | Yes | Language | IETF BCP 47 language code
```

Comment:
It is kind of a bummer that you have to repeat the same text + language snippet for every translated field.  In the spirit of DRY, would you consider introduced an `Array<Localized String>` type (or something equivalent) and defined the fields of `Localized String` above in the "Field types" section?

Reply:
Yeah, we went a few rounds on this but ended up going with what seems to be the overall style of the spec. This could always be changed without it being breaking.

Reply:
I wanted to recommend that you also update `language` description to say:

> An IETF BCP 47 language code that MUST match one of values specified in the `languages` field in `system_information.json`.

However, it seems silly to repeatedly duplicate that guidance for every translatable field.

Reply:
Yes I agree. I have played around with some varieties of this but all felt clunky to me and kind of collided with the style of the rest of the spec. However, if you have a concrete proposal on how this would look would you mind drafting it up?

Reply:
E.g. the notation Array<Type> is not used elsewhere in the spec

Reply:
Here's an example of what the syntax could look like:

https://github.com/MobilityData/gbfs/compare/master...bdferris-v2:gbfs:localized-string?expand=1
https://github.com/bdferris-v2/gbfs/blob/2b98e0280279ce705de746a3366ebf6795209dfc/gbfs.md

Reply:
I'm not against this, but it will take a while to update the proposal - and the vote may have to be postponed.

@mplsmitch what do you think about this style? It certainly makes things much less verbose.

Reply:
On the other hand - changing this separately will not be breaking...

Reply:
Less verbose is better but maybe we should try to pass this as-is, and then open a separate vote on changing to the localized string format. It seems like it would be an easy sell with some example JSON showing how much lighter it would be.